commit 830773130aa1b776eb0674942d630e0ebd81299d
Author: Marc Kleine-Budde <mkl@blackshift.org>
Date:   Sat Sep 12 17:29:59 2015 +0200

    net: rtl8192cu: Fix off-by-one warning
    
    ../drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_rf6052.c: In function ‘PHY_RFShadowRefresh’:
    ../drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_rf6052.c:1020:37: warning: iteration 63 invokes undefined behavior [-Waggressive-loop-optimizations]
        RF_Shadow[eRFPath][Offset].Value = 0;
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~
    ../drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_rf6052.c:1018:3: note: within this loop
       for (Offset = 0; Offset <= RF6052_MAX_REG; Offset++)
       ^~~
    
    Signed-off-by: Marc Kleine-Budde <mkl@blackshift.org>
    Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>

diff --git a/drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_rf6052.c b/drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_rf6052.c
index cf46969379a8..5995a4b85dc5 100755
--- a/drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_rf6052.c
+++ b/drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_rf6052.c
@@ -1015,7 +1015,7 @@ PHY_RFShadowRefresh(
 
 	for (eRFPath = 0; eRFPath < RF6052_MAX_PATH; eRFPath++)
 	{
-		for (Offset = 0; Offset <= RF6052_MAX_REG; Offset++)
+		for (Offset = 0; Offset < RF6052_MAX_REG; Offset++)
 		{
 			RF_Shadow[eRFPath][Offset].Value = 0;
 			RF_Shadow[eRFPath][Offset].Compare = _FALSE;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1093_830773130aa1b776eb0674942d630e0ebd81299d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1093_830773130aa1b776eb0674942d630e0ebd81299d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
